src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -301,6 +302,7 @@ namespace AppInstaller::Manifest
 
                 std::set<std::string> commandAliasSet;
                 std::set<std::string> relativeFilePathSet;
+                const bool isPortable = installer.NestedInstallerType == InstallerTypeEnum::Portable;

```

Comment:
nit: I don't think the const is needed here
```suggestion
                bool isPortable = installer.NestedInstallerType == InstallerTypeEnum::Portable;

```